### PR TITLE
[DOC-5471][DOC-4766] Document provisioned-rate store field

### PR DESCRIPTION
**Pull request by mdlinville**
[DOC-5471]
[DOC-4766]

Relates to https://github.com/cockroachdb/cockroach/pull/86063

## Previews
[v22.2/cockroach-start.md](https://deploy-preview-15665--cockroachdb-docs.netlify.app/docs/v22.2/cockroach-start.html)

## Tests
- [x] Local build
- [x] Linkcheck

### CODE REVIEW

**Review by sumeerbhola — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15665)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @mdlinville)

___
*[`v22.2/cockroach-start.md` line 212 at r1](https://reviewable.io/reviews/cockroachdb/docs/15665#-NHUof0z933Nb8rLwspM:-NHUof1-3z7YeWIZg0V-:b-g95831) ([raw file](https://github.com/cockroachdb/docs/blob/de49388ea1155e84b2a86e415435e7b91ab9b55d/v22.2/cockroach-start.md#L212)):*
> ```Markdown
> <a name="store-size"></a> `size` | The maximum size allocated to the node. When this size is reached, CockroachDB attempts to rebalance data to other nodes with available capacity. When no other nodes have available capacity, this limit will be exceeded. Data may also be written to the node faster than the cluster can rebalance it away; as long as capacity is available elsewhere, CockroachDB will gradually rebalance data down to the store limit.<br><br> The `size` can be specified either in a bytes-based unit or as a percentage of hard drive space (notated as a decimal or with `%`), for example: <br><br>`--store=path=/mnt/ssd01,size=10000000000 ----> 10000000000 bytes`<br>`--store=path=/mnt/ssd01,size=20GB ----> 20000000000 bytes`<br>`--store=path=/mnt/ssd01,size=20GiB ----> 21474836480 bytes`<br>`--store=path=/mnt/ssd01,size=0.02TiB ----> 21474836480 bytes`<br>`--store=path=/mnt/ssd01,size=20% ----> 20% of available space`<br>`--store=path=/mnt/ssd01,size=0.2 ----> 20% of available space`<br>`--store=path=/mnt/ssd01,size=.2 ----> 20% of available space`<br><br>**Default:** 100%<br><br>For an in-memory store, the `size` field is required and must be set to the true maximum bytes or percentage of available memory, for example:<br><br>`--store=type=mem,size=20GB`<br>`--store=type=mem,size=90%`<br><br><strong>Note:</strong> If you use the `%` notation, you might need to escape the `%` sign, for instance, while configuring CockroachDB through `systemd` service files. For this reason, it's recommended to use the decimal notation instead.
> <a name="fields-ballast-size"></a> `ballast-size` | Configure the size of the automatically created emergency ballast file. Accepts the same value formats as the [`size` field](#store-size). For more details, see [Automatic ballast files](cluster-setup-troubleshooting.html#automatic-ballast-files).<br><br>To disable automatic ballast file creation, set the value to `0`:<br><br>`--store=path=/mnt/ssd01,ballast-size=0`
> <a name="store-partition-rate"></a> `partition-rate` | A mapping of a store name to a bandwidth limit, expressed in bytes per second. A value of `0` (the default) represents unlimited bandwidth. For example: <br /><br />`--store=partition-rate=disk-name=/mnt/ssd01=200`<br /><br />This value overrides the [`kv.store.admission.provisioned_bandwidth` cluster setting](cluster-settings.html#settings).
> ```

I don't understand the term partition-rate. What I had added was provisioned-rate: see the release note in  https://github.com/cockroachdb/cockroach/pull/86063.
Also the code comment may be helpful:
```
//   - provisioned-rate=disk-name=<disk-name>[:bandwidth=<bandwidth-bytes/s>] The
//     provisioned-rate can be used for admission control for operations on the
//     store. The bandwidth is optional, and if unspecified, a cluster setting
//     (kv.store.admission.provisioned_bandwidth) will be used.
```

**Review by mdlinville — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15665)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @sumeerbhola)

___
*[`v22.2/cockroach-start.md` line 212 at r1](https://reviewable.io/reviews/cockroachdb/docs/15665#-NHUof0z933Nb8rLwspM:-NHVlUd6AHIJfgtwR5Y3:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/de49388ea1155e84b2a86e415435e7b91ab9b55d/v22.2/cockroach-start.md#L212)):*
<details><summary><i>Previously, sumeerbhola wrote…</i></summary><blockquote>

I don't understand the term partition-rate. What I had added was provisioned-rate: see the release note in  https://github.com/cockroachdb/cockroach/pull/86063.
Also the code comment may be helpful:
```
//   - provisioned-rate=disk-name=<disk-name>[:bandwidth=<bandwidth-bytes/s>] The
//     provisioned-rate can be used for admission control for operations on the
//     store. The bandwidth is optional, and if unspecified, a cluster setting
//     (kv.store.admission.provisioned_bandwidth) will be used.
```
</blockquote></details>

Done.

**Review by sumeerbhola — APPROVED** [review]
:lgtm: 

> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/15665#-:-NI3jKgu3eg4JB3zRLfb:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @mdlinville)

File: v22.2/cockroach-start.md [hunk 1]
```diff
@@ -209,6 +209,7 @@ Field | Description
...
+<a name="store-provisioned-rate"></a> `provisioned-rate` | A mapping of a store name to a bandwidth limit, expressed in bytes per second. This constrains the bandwidth used for [admission control](admission-control.html) for operations on the store. The disk name is separated from the bandwidth value by a colon (`:`). A value of `0` (the default) represents unlimited bandwidth. For example: <br /><br />`--store=provisioned-rate=disk-name=/mnt/ssd01:200`<br /><br />If the bandwidth value is omited, bandwidth is limited to the value of the  [`kv.store.admission.provisioned_bandwidth` cluster setting](cluster-settings.html#settings). <strong>Modify this setting only in consultation with your <a href="https://support.cockroachlabs.com/hc/en-us">support team</a>.</strong>
```
Comment (taroface) [inline v22.2/cockroach-start.md:212]: Suggesting for consistency with the other rows

```suggestion
<a name="store-provisioned-rate"></a> `provisioned-rate` | A mapping of a store name to a bandwidth limit, expressed in bytes per second. This constrains the bandwidth used for [admission control](admission-control.html) for operations on the store. The disk name is separated from the bandwidth value by a colon (`:`). A value of `0` (the default) represents unlimited bandwidth. For example: <br /><br />`--store=provisioned-rate=disk-name=/mnt/ssd01:200`<br /><br />**Default:** 0<br /><br />If the bandwidth value is omited, bandwidth is limited to the value of the  [`kv.store.admission.provisioned_bandwidth` cluster setting](cluster-settings.html#settings). <strong>Modify this setting only in consultation with your <a href="https://support.cockroachlabs.com/hc/en-us">support team</a>.</strong>
```

**Review by taroface — APPROVED** [review]
LGTM plus one suggestion

**Pull request merged**